### PR TITLE
Ensures that the directory path is disabled for editing projects (unless the user is a system administrator)

### LINKED ISSUE #1046: The `Directory Path` field should not be writable after it is approved
In Tigerdata, the edit form displays two fields `Project Directory` which is read only, and `Directory Path` which is writable. Editing the `Directory Path` and submitting the form updates both fields. Since both fields seem to be technically displaying the same information, It would make most sense to only display one and choose whether it is read only or read/write.

Acceptance Criteria:

- [ ] Data Sponsor should view the path, but not edit it
- [ ] Data Manager should view the path, but not edit it
- [ ] Data user should view the path, but not edit it
- [ ] System Administrator should view and edit  the path

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #1046

### CODE REVIEW

File: spec/system/project_spec.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
 RSpec.describe "Project Page", connect_to_mediaflux: true, type: :system  do
   let(:sponsor_user) { FactoryBot.create(:project_sponsor, uid: "pul123") }
   let(:sysadmin_user) { FactoryBot.create(:sysadmin, uid: "puladmin") }
+  let(:superuser) { FactoryBot.create(:superuser, uid: "root") }
```
Comment (carolyncole) [inline spec/system/project_spec.rb:8]: @jrgriffiniii should this line be removed since it is unused?

**Review by carolyncole — DISMISSED** [review]
@jrgriffiniii It looks like you are not testing your code change anywhere, maybe you forgot to commit some changes?

**Comment by kelynch** [comment]
It looks like this branch needs to be updated with the latest changes from main.  When I ran the test suite, `/spec/system/project_roles_spec.rb:209` produced an error.

**Review by kelynch — CHANGES_REQUESTED** [review]
It looks like this branch needs to be updated with the latest changes from main. When I ran the test suite, /spec/system/project_roles_spec.rb:209 produced an error.

**Review by carolyncole — APPROVED** [review]
Thanks @jrgriffiniii.  Looks great!

**Pull request merged**